doc/related_projects.rst

```diff
@@ -302,3 +302,20 @@ Domain specific packages
 
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
+
+Translations of scikit-learn documentation
+------------------------------------------
+
+The only official documentation for scikit-learn is the current English
+version at https://scikit-learn.org. However, some translations are available aiming to improve scikit-learn
```

Comment:
Although we don't enforce 80 chars in rst files, I think we try (or should try) to not respect it.
Even if there are many violations within this file itself, it's never too late to start :)